pom.xml

```diff
@@ -43,7 +43,7 @@
 		<dependency>
 			<groupId>org.spdx</groupId>
 			<artifactId>spdx-tools</artifactId>
-			<version>2.2.2</version>		
+			<version>2.2.3</version>		
```

Comment:
This hasn't been pushed to Maven yet, so it will cause a build failure.  No change needed, I'll publish a 2.2.3 version once I add the end-index.

Reply:
Ok